drivers/slipdev/slipdev.c

```diff
@@ -355,8 +440,82 @@ static const netdev_driver_t slip_driver = {
 #endif
 };
 
+#if IS_USED(MODULE_SLIPDEV_STDIO)
+static uint16_t _fcs_part(const uint8_t * data, size_t len) {
+    uint16_t fcs = SPECIAL_INIT_FCS;
+    for (size_t i = 0; i < len; ++i)
+    {
+        fcs = (fcs >> 8) ^ FCS_LOOKUP[(fcs ^ data[i]) & 0xff];
```

Comment:
This looks a lot like `crc16_ccitt_kermit_update()`

Reply:
You know whats funny? I wrote the unittest for that function.